app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -72,6 +74,7 @@ class HomeFragment : Fragment() {
 
     private fun initTiles() = with (tileContainer) {
         val homeTiles = mutableListOf<HomeTile>()
+        homeTiles.addAll(HomeTilesManager.getCustomHomeTilesList(context))
```

Comment:
nit: this file is hard to read because there isn't much WS between lines. I think it'd help if you added a whitespace below this `getCustomTiles` event, to separate it from constructing the default tiles.